.github/dependabot.yml

```diff
@@ -19,10 +19,10 @@ updates:
       - dependency-name: hashicorp/setup-terraform
       - dependency-name: mxschmitt/action-tmate
       - dependency-name: step-security/harden-runner
-      # # Managed by cisagov/cyhy-db
```

Comment:
This is an indication that you need to run `gh extension upgrade --all`.

Reply:
I think @felddy did that at some point after he created this new repo, but it will be good for him to confirm.

Reply:
It was definitely @felddy [who made the change](https://github.com/cisagov/cyhy-db/blame/first-commits/.github/workflows/build.yml#L65).  Maybe the version wasn't 3.12 yet then.  Soon it will be 3.13, which is already out.

In any event, the change can and should be reverted now.

Reply:
Updated `gh` extension.  Do we need a change here?

Reply:
Nope.